src/utils/index.ts

```diff
@@ -142,3 +143,27 @@ export const tokenRequest = async (query: string, silenceRequestError: boolean)
 		throw new Error(`HTTP error.`)
 	}
 }
+
```

Comment:
Could you write a unit test for this logic, testing pass and fail scenarios.

Thanks.

Reply:
@nicktaras Unit tests added